pages/1.13/security/ent/secure-compute-profiles/index.md

```diff
@@ -3,75 +3,72 @@ layout: layout.pug
-
-Seccomp filters are defined in seccomp profiles using JSON formatting similar to this [sample seccomp profile](http://mesos.apache.org/documentation/latest/examples/seccomp_default.json).
-
-To support secure command execution for both UCR and Docker containers, the seccomp profile must include both the `pivot_root` system call and the `chroot` system call, in the `names` array. For example:
-
-```bash
+```
 {
-  "names": [
-    "chroot",
-    "pivot_root"
-  ],
-  "action": "SCMP_ACT_ALLOW",
-  "args": [],
-  "comment": "",
-  "includes": {
-    "caps": [
-      "CAP_SYS_CHROOT"
-    ]
-  },
-  "excludes": {}
-},
+  "id": "/mesos-seccomp-app",
+  "cmd": "sleep 1000",
+  "cpus": 0.5,
+  "mem": 32,
+  "container": {
+    "type": "MESOS",
+    "linuxInfo": {
+      "seccomp": {
+        "unconfined" : true
+      }
+    }
+  }
+}
 ```
+By configuring a service definition with an `unconfined` seccomp, the container will NOT run under seccomp. This will allow this container to execute any syscall that might have been restricted by the default seccomp profile.
 
-# Configuring the agent for secure computing mode
-You can add the DC/OS seccomp isolator `linux/seccomp` to an agent by setting the `--isolation` or `--mesos-seccomp-enabled` agent configuration option. You must have `root` privileges to install the seccomp isolator and seccomp filters to avoid the issues involved when using a user account without root-level privileges.
-
-In addition to the `--isolation` or `--mesos-seccomp-enabled` option, you should configure the agent using the following settings:
-- Use `--seccomp_config_dir` to specify the path to the directory containing seccomp profiles.
-- Use `--mesos_seccomp_profile_name` to specify the default seccomp profile you want to apply by default for all DC/OS containers. The profile name must be relative to the `--seccomp_config_dir` path. If you don't specify this option, no default seccomp profile is defined and, therefore, no profile is applied by default.
-
-The following illustrates how to start a DC/OS agent with secure computing mode enabled:
+It is also possible to have a service definition run under a different seccomp profile other than the default. This is accomplished by specifying the profile name to seccomp definition for the service definition.
 
-```bash
-sudo mesos-agent --master=<master ip> --ip=<agent ip> --work_dir=/var/lib/mesos --isolation=linux/seccomp --seccomp_config_dir=/etc/mesos/seccomp --seccomp_profile_name=default.json
 ```
+{
+  "id": "/mesos-seccomp-app",
+  "cmd": "sleep 1000",
+  "cpus": 0.5,
+  "mem": 32,
+  "container": {
+    "type": "MESOS",
+    "linuxInfo": {
+      "seccomp": {
+        "profileName" : "relaxed.json",
```

Comment:
nit: profile_name https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto#L3225

Reply:
@kvish `profileName` is correct; this is not a payload that's simply routed to to the TaskInfo. This is typed as of https://github.com/mesosphere/marathon/blob/master/docs/docs/rest-api/public/api/v2/types/linux.raml#L9

Reply:
```suggestion
        "profileName": "relaxed.json",
```